package com.kh.example.chap01_exception.run;
import java.io.IOException;

import com.kh.example.chap01_exception.controller.ExceptionController;

public class Run {
	public static void main(String[] args) throws IOException {
		

		ExceptionController ec = new ExceptionController();
		ec.method1();
		System.out.println("정상적으로 종료됨...");
	}	
}
